doc/build-windows.md

```diff
@@ -47,9 +47,9 @@ This means you cannot use a directory that is located directly on the host Windo
 
 Build using:
 
-    gmake -C depends HOST=x86_64-w64-mingw32  # Use "-j N" for N parallel jobs.
+    gmake -C depends HOST=x86_64-w64-mingw32  # Append "-j N" for N parallel jobs.
```

Comment:
review note: `gmake` seems fine with placing `-j` earlier, so technically this change isn't necessary, but it's probably better to be consistent anyway.

Reply:
Good point. Same for `ctest`, I think.